### ISSUE TITLE
Nikon Z 8 not able to edit in darkroom in Darktable 5.0.0

### ISSUE THREAD

**Issue by ivanwills** [darktable-org/darktable#18092]
### Describe the bug

When I try to edit an RAW image created on a Nikon Z 8 I get a message 

> image `...nef' could not be loaded file appears corrupt

All images are able to be displayed in lightroom and the files are available on disk.

Originally the message appeared in darkroom it self but now they only appear as a notice in lightroom.

### Steps to reproduce

Double click on image and darkroom should open up (works with my older Nikon D500 images)

### Expected behavior

show the file to edit

### Logfile | Screenshot | Screencast

```
     3.9884 [rawspeed] 2024-11-26-0027371-DSC_4997.nef corrupt: rawspeed::RawImage rawspeed::RawDecoder::decodeRaw(), line 338: rawspeed::Buffer::size_type rawspeed::ByteStream::check(rawspeed::Buffer::size_type) const, line 64: Out of bounds access in ByteStream
/usr/lib/x86_64-linux-gnu/gvfs/libgvfscommon.so: undefined symbol: g_task_set_static_name
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgioremote-volume-monitor.so
/usr/lib/x86_64-linux-gnu/gvfs/libgvfscommon.so: undefined symbol: g_task_set_static_name
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgioremote-volume-monitor.so
/usr/lib/x86_64-linux-gnu/gvfs/libgvfscommon.so: undefined symbol: g_task_set_static_name
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgvfsdbus.so
/usr/lib/x86_64-linux-gnu/gio/modules/libdconfsettings.so: undefined symbol: g_assertion_message_cmpint
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libdconfsettings.so
     6.7537 [rawspeed] 2024-11-26-0027377-DSC_5003.nef corrupt: rawspeed::RawImage rawspeed::RawDecoder::decodeRaw(), line 338: rawspeed::Buffer::size_type rawspeed::ByteStream::check(rawspeed::Buffer::size_type) const, line 64: Out of bounds access in ByteStream

** (darktable:1832565): WARNING **: 19:00:44.005: atk-bridge: get_device_events_reply: unknown signature
```

### Where did you obtain darktable from?

downloaded from www.darktable.org

### darktable version

5.0.0 (Appimage)

### What OS are you using?

Linux

### What is the version of your OS?

Ubuntu 24.10

### Describe your system?

Dell XPS 32GB Ram (2019)

### Are you using OpenCL GPU in darktable?

I dont know

### Please provide additional context if applicable. You can attach files too, but might need to rename to .txt or .zip

This did happen in 4.8.1 but at that time Nikon Z 8 weren't fully supported according to the documentation. (I would upload a sample but even compressed they are all bigger than 25MB)

**Comment by ralfbrown** [comment]
Z 8 still isn't fully supported - you must use "lossless compression", the "HE" and "HE*" compression modes are not supported.  If you are *not* using one of the High Efficiency compression modes, we'll need an image file which has the problem for further investigation.

**Comment by ivanwills** [comment]
I don't believe I'm using a HE format (worked that one out when I tried it when I first got the camera) 
[DSC_1558.nef.gz](https://github.com/user-attachments/files/18267802/DSC_1558.nef.gz)
The attached is a sample image from my camera.

**Comment by ralfbrown** [comment]
Exiftool reports the file to be using HE* compression.
